zerver/lib/import_realm.py

```diff
@@ -256,14 +257,14 @@ def fix_message_rendered_content(realm: Realm,
             # We don't handle alert words on import from third-party
             # platforms, since they generally don't have an "alert
             # words" type feature, and notifications aren't important anyway.
-            realm_alert_words = dict()  # type: RealmAlertWords
+            realm_alert_words_automaton = ahocorasick.Automaton()  # type: ahocorasick.Automaton
```

Comment:
`do_render_markdown` doesn't take in a `None` only `render_markdown` does. I believe `do_render_markdown` was used as opposed to `render_markdown` since the `sender` is fetched from a `sender_map`  instead of `get_user_profile_by_id(message.sender_id)` from `render_markdown`. So i decided to go with the flow and created a empty automaton since this looked like a one off case

Reply:
I think we should just make `do_render_markdown` support taking `None` for this argument by just tagging that argument as `Optional[ahocorasick.Automaton]`; if the goal is to not have alert words processing happen, that seems like a cleaner way to communicate that than a half-initialized automaton.

Reply:
This makes sense. automaton having None and having it as something other than ahocorasick would mean the same thing. I have modified `do_render_markdown` to take in a `None`. The automaton would be build only in `alert_words.py` and it would return None from there in case it is not of type Ahocorasick